openpaas-james/apps/distributed/README.md

```diff
@@ -0,0 +1,52 @@
+# openpaas-james Distributed server
+
+This server is the distributed version of James relying on:
+
+* Cassandra
+* S3 API-like object storage
+* ElasticSearch
+* RabbitMQ
+* Tika (optional)
+
+## Build
+
+To build the distributed server:
+
+```
+mvn clean install
+mvn compile com.google.cloud.tools:jib-maven-plugin:2.7.0:dockerBuild
+```
+
+## Run
+
+### Run manually
+
+You need to start first all the other softwares the distributed James is relying on:
+
+```
+docker run -d --name=cassandra cassandra:3.11.3
+
+docker run -d --name=rabbitmq rabbitmq:3.8.1-management
+
+docker run -d --env 'REMOTE_MANAGEMENT_DISABLE=1' --env 'SCALITY_ACCESS_KEY_ID=accessKey1' --env 'SCALITY_SECRET_ACCESS_KEY=secretKey1' --name=s3 zenko/cloudserver:8.2.6
+
+docker run -d --name=elasticsearch --env 'discovery.type=single-node' docker.elastic.co/elasticsearch/elasticsearch:6.3.2
+
+docker run -d --name=tika apache/tika:1.24 #Optional
+```
+
+Then you can finally start the James distributed server:
+
+```
+docker run -d --hostname HOSTNAME -p "25:25" -p 80:80 -p "110:110" -p "143:143" -p "465:465" -p "587:587" -p "993:993" -p "8000:8000" --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link s3:s3.docker.test --name james -t linagora/openpaas-james-distributed
```

Comment:
Use networks, not links please

Reply:
I thought that was for just docker-compose, my bad :)